src/app/index.ts

```diff
@@ -289,12 +290,17 @@ module.exports = class extends yo {
       this.project = {
         folder: this.options.output || answerForName.name || this.options.name,
         name: this.options.name || answerForName.name,
-        host: this.options.host || answerForHost.host,
         projectType: _.toLower(this.options.projectType) || _.toLower(answerForProjectType.projectType),
         isManifestOnly: isManifestProject,
         isExcelFunctionsProject: isExcelFunctionsProject,
-        scriptType: answerForScriptType.scriptType ? answerForScriptType.scriptType : this.options.ts ? typescript : javascript
       };
+      this.project.host = answerForHost.host ? answerForHost.host 
```

Comment:
Why not leave this in the block using "host: answerforHost.host . . ." with everything else (you don't need to use "this.project.")?

Reply:
They set projectType inside the block as well.  It's a big deal but I would add 'const projType = _.toLower(this.options.projectType) || _.toLower(answerForProjectType.projectType)' before the block then create everything inside the block using the projType variable.